### DISCUSSION TITLE
GEOL_LEG & BKFL_LEG

### DISCUSSION THREAD

**Discussion by richard-sisson** [australiangeomechanics/agsformat#64] [Q&A] [answered]
Hi,

I see that all of the GEOL_LEG & BKFL_LEG codes from AGS UK have been included in the new AU lists.

I note that the soil classification based codes (i.e. CI) have been removed.  

Is there any guidance on the use of these new codes, in particular is it expected that CLAY is used as a replacement for CI, or is the user now free to use any code?

Also, is there any guidance on the mapping of old codes to new codes, such as CI maps to 201 (CLAY)?

Thanks
Richard

**Comment by richard-sisson** [comment]
Is AU998 in GEOL_LEG correct? 
![image](https://user-images.githubusercontent.com/10456433/187816510-6dff4d55-0e85-4170-8eaf-fc945c3d0d87.png)

**Comment by SolidRockProgrammer** [comment, marked as answer]
There are not new and old codes.  AGS 3.1 RTA 1.1 is not a direct predecessor of AGS 4.1.1 AU x. We are not mapping anything back to AGS 3.1 RTA 1.1.
AGS 4.1.1 AU uses the standard AGS format codes where they apply, and we added AU### with our own prefix to allow us to maintain our own list or additions, not clashing with std AGS in the future.

Some humor was included;)

**Comment by richard-sisson** [comment]
Thanks for the clarification Phil, and glad that I don't have to consider this 'earthy' material :)

Just to confirm, the following codes should be considered unique:
102 (MADE GROUND) & AU102 (FILL)
103 (Bituminous Material) & AU101 (ASPHALTIC CONCRETE)
998 (No Recovery) & AU999 (NO CORE - is this the same as CORE LOSS?)

Thanks
Richard

**Reply by SolidRockProgrammer** [reply]
The codes listed in sheet ABBR exist in AGS 4.1.1 AU 1.2.  If one can't find a reasonable existing code for teh thing they have, one can make a new code.  We made some new codes because the terms used in Australia are different from the UK, even if they mean the the same thing for someone with international exposure.

**Comment by SolidRockProgrammer** [comment]
We added codes to suit Australian practice and common Australian terminology. I don't get your question, but there isn't much more I can say.

Sent by mobile - Phil Wade
________________________________
